app/src/test/java/org/simple/clinic/summary/PatientSummaryScreenControllerTest.kt

```diff
@@ -381,6 +381,61 @@ class PatientSummaryScreenControllerTest {
     verify(medicalHistoryRepository).save(eq(updatedMedicalHistory), any())
   }
 
+  @Test
+  @Parameters(method = "params for editing blood pressures")
+  fun `when blood pressure is clicked for editing, only those recorded in a given period must be edited`(
+      bpEditableFor: Duration,
+      bloodPressureMeasurement: BloodPressureMeasurement,
+      shouldBeEditable: Boolean
+  ) {
+    val config = PatientSummaryConfig(numberOfBpPlaceholders = 0, bpEditableFor = bpEditableFor)
+    configEmitter.onNext(config)
+
+    uiEvents.onNext(PatientSummaryBpClicked(bloodPressureMeasurement))
+
+    if(shouldBeEditable) {
+      verify(screen).showBloodPressureUpdateSheet(bloodPressureMeasurement.uuid)
+    } else {
+      verify(screen, never()).showBloodPressureUpdateSheet(any())
+    }
+  }
+
+  @Suppress("Unused")
+  private fun `params for editing blood pressures`(): List<List<Any>> {
```

Comment:
@vinaysshenoy this function doesn't look it was written in readable manner for people :p There are lots of magic values like `duration1`, `duration2` which I'm having trouble understanding. Can this be improved?

Reply:
Seems to be a duplicate comment. Closing this.